test/ssl-tests/27-ticket-appdata.conf.in

```diff
@@ -0,0 +1,57 @@
+# -*- mode: perl; -*-
+# Copyright 2016-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+## Test session ticket app data
+
+use strict;
+use warnings;
+
+package ssltests;
+
+
+our @tests = (
+    {
+	"name" => "session-ticket-app-data",
+	"client" => {
+	    "MaxProtocol" => "TLSv1.2",
```

Comment:
Why only TLSv1.2? Shouldn't we also test TLSv1.3?

Reply:
Yeah we should, this was originally written against 1.1.0...